app/default/routes.py

```diff
@@ -249,17 +249,19 @@ def submit_application():
         json=payload,
     )
     submitted = submission_response.json()
-    application_id = submitted.get("id")
-    application_reference = submitted.get("reference")
-    application_email = submitted.get("email")
-    if not application_reference or not application_email:
+    if submission_response.status_code != 201 or not submitted.get(
```

Comment:
Have we checked the API definitely returns a `201` in the success case?

Reply:
Yes I checked that

Reply:
https://github.com/communitiesuk/funding-service-design-application-store/blob/main/openapi/api.yml#L180